src/test_l1_dsFPD.c

```diff
@@ -393,13 +400,16 @@ void test_l1_dsFPD_positive_dsSetFPState (void)
  * |06|Terminate with dsFPTerm()| |dsERR_NONE|Ensure the system is terminated|
```

Comment:
Correct the description in test procedure as well

Reply:
Done